src/components/dialog/dialog.tsx

```diff
@@ -145,6 +145,7 @@ export class Dialog {
                 class={{
                     'mdc-dialog': true,
                     'full-screen': !!this.fullscreen,
+                    popup: !!window.opener,
```

Comment:
Hmm… [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/opener) for `window.opener` says:

> The [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window) interface's `opener` property returns a reference to the window that opened the window, either with [open()](https://developer.mozilla.org/en-US/docs/Web/API/Window/open), or by navigating a link with a [target](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) attribute.
>
> In other words, if window `A` opens window `B`, `B.opener` returns `A`.

I think that means that if anyone uses the "open in a new window" button (see screenshot), any full screen dialogs in the new tab will get this behaviour… 🤔

Reply:
> > Why would we even want to do that?
> 
> It's for the desktop client. Rendering a dialog inside the action pad does not make much sense 😄

Ah! Of course! Yeah, then we really need this 👍

That also shifts my opinion on the value of this vs the downside that I mentioned in my comment on the code…

But, I don't think we should apply it so blindly here in lime-elements. I think this needs to be controlled by the consumer somehow… property, css-variable, css-class 🤷‍♂️

Reply:
Yeah, sure, we can maybe add a CSS property or something

Reply:
> I think that means that if anyone uses the "open in a new window" button (see screenshot), any full screen dialogs in the new tab will get this behaviour… 

I tried this and the `opener` was not set, but I agree that the solution I did not feel 100% fool proof

Reply:
Huh, interesting… 🤔

Still, if possible within reason, it would be nice to have the consumer control it 👍

Reply:
> But, I don't think we should apply it so blindly here in lime-elements. I think this needs to be controlled by the consumer somehow… property, css-variable, css-class 🤷‍♂️

I agree. I think a css variable would be nice, safe, and wise